apps/frontend/src/components/global/UserModal.vue

```diff
@@ -370,22 +373,24 @@ export default class UserModal extends Vue {
   }
 
   getAPIKeys() {
-    this.apiKeyTableLoading = true;
-    axios
-      .create()
-      .get<IApiKey[]>(`/apikeys`, {params: {userId: this.user.id}})
-      .then(({data}) => {
-        this.apiKeys = data;
-      })
-      .catch((error) => {
-        if (error.response) {
-          if (error.response.status === 403) {
-            this.apiKeysDisabled = true;
-            return;
+    if (this.apiKeysEnabled) {
+      this.apiKeyTableLoading = true;
+      axios
+        .create()
+        .get<IApiKey[]>(`/apikeys`, {params: {userId: this.user.id}})
+        .then(({data}) => {
+          this.apiKeys = data;
+        })
+        .catch((error) => {
+          if (error.response) {
+            if (error.response.status === 403) {
```

Comment:
we should make the snackbar thing pop up regardless of which particular error response code we get (i.e. get rid of this nested if, the top level one is sufficient)

Reply:
fixed